### PR TITLE
Add Paralogues plugin to web VEP (e113)

### PR DESCRIPTION
**Pull request by nuno-agostinho**
[ENSVAR-6290](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-6290)

## Testing

Test in my sandbox:
- Documentation changes:
    - [Web VEP input docs](http://wp-np2-11.ebi.ac.uk:6070/info/docs/tools/vep/online/input.html#freq)
    - [What's new in VEP](http://wp-np2-11.ebi.ac.uk:6070/info/docs/tools/vep/online/download.html#new)
- [Web VEP](http://wp-np2-11.ebi.ac.uk:6070/Multi/Tools/VEP?db=core)
- [Results example](http://wp-np2-11.ebi.ac.uk:6070/Homo_sapiens/Tools/VEP/Results?from=1;size=12;tl=NmXCXfSwVTsg7glP-23;to=12)

In web VEP, you can test with the following variants (not all chromosomes are available in my sandbox):
```
1 18108132 . A T
11 99556218 . G C
19 110682 . T G
20 87737 . C G
20 325925 . A T
22 15528369 . T G
22 15528390 . T G
22 15528213 . A C
22 15528195 . T G
X 630901 . A T
X 630907 . T C
X 1190999 . T C
```

### CODE REVIEW

**Review by nakib103 — APPROVED** [review]
Looks good !~ thanks @nuno-agostinho

File: tools_hive/conf/vep_plugins_hive_config.txt [hunk 1]
```diff
@@ -158,4 +158,11 @@
       "file=[[ENSEMBL_VEP_PLUGIN_DATA_DIR]]/ClinPred/ClinPred_hg38_sorted_tabbed.tsv.gz"
     ]
   },
+  Paralogues => {
+    "params" => [
+      "matches=[[ENSEMBL_VEP_PLUGIN_DATA_DIR]]/Paralogues/Paralogues.pm_homo_sapiens_113_ClinVar_20240819.tsv.gz",
```
Comment (imransl) [inline tools_hive/conf/vep_plugins_hive_config.txt:163]: @nuno-agostinho Is this file already available?

Comment (nakib103) [reply tools_hive/conf/vep_plugins_hive_config.txt:163]: Hi @imransl,

yes, the Paralogues plugin data should have been handed over for e113 - 
```
$ ls /nfs/production/flicek/ensembl/variation/enseweb-data_tools/grch38/e113/vep/plugin_data/Paralogues/
Paralogues.pm_homo_sapiens_113_ClinVar_20240819.tsv.gz  Paralogues.pm_homo_sapiens_113_ClinVar_20240819.tsv.gz.tbi
```

Comment (imransl) [reply tools_hive/conf/vep_plugins_hive_config.txt:163]: @nakib103 Okay I can see it there now, but it has been added well after we had finished the syncing. Please let us know when new files like these become available again to sync them to the live directories.

Comment (nakib103) [reply tools_hive/conf/vep_plugins_hive_config.txt:163]: @imransl, yeah it was supposed to go in for e114 but we then added a bit earlier. I think it got lost in the process to let web team know about the new data. Sorry about that!

Comment (imransl) [reply tools_hive/conf/vep_plugins_hive_config.txt:163]: No problem. The directories have been synced now, so I will merge this PR.

**Pull request merged**